modules/Collections/Controller/Admin.php

```diff
@@ -105,7 +105,12 @@ public function collection($name = null) {
         // get field templates
         $templates = [];
 
-        foreach ($this->app->helper('fs')->ls('*.php', 'collections:fields-templates') as $file) {
+        $templatesPath = 'collections:fields-templates';
+        if ($customTemplatesPath = $this->path('#config:collections/templates')) {
```

Comment:
why not allow both? collect the paths in an array and then do a loop over the folders

Reply:
This was intentional to avoid naming conflicts and to have the ability to override the default template(s). If I choose to use custom templates I can still copy the core collection template to my template folder.